SampleNHAppSwift/SampleNHAppSwift/AppDelegate.swift

```diff
@@ -1,22 +1,44 @@
-//
-//  AppDelegate.swift
-//  SampleNHAppSwift
-//
-//  Created by Matthew Podwysocki on 6/30/20.
-//  Copyright © 2020 Matthew Podwysocki. All rights reserved.
-//
+//----------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//----------------------------------------------------------------
 
 import UIKit
+import WindowsAzureMessaging
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
+    var connectionString: String?
+    var hubName: String?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+        
+        if let path = Bundle.main.path(forResource: "DevSettings", ofType: "plist") {
```

Comment:
Should we warn them if those values are not in there that this won't work?

Reply:
Added log and exit(-1)